resources/views/helper.php

```diff
@@ -21,15 +21,12 @@
  */
 
 <?php foreach ($namespaces_by_extends_ns as $namespace => $aliases) : ?>
-    <?php if ($namespace == '\Illuminate\Database\Eloquent') :
-        continue;
-    endif; ?>
```

Comment:
Why is this removed?

Reply:
It was originally filtered out because it's not a facade so it doesn't need to be there but all non facade classes are filtered out already.

Reply:
Ah, nice, thank you 👍